examples/nanocoap_server/README.md

```diff
@@ -0,0 +1,142 @@
+nanocoap server example
+========================
+
+This application is meant to get you started with implementing a CoAP server on RIOT.
+It uses the GNRC network stack through RIOT's
+[sock API](http://doc.riot-os.org/group__net__sock.html).
+
+Usage
+=====
+
+To try out the server on native, compile it with
+
+```
+$ make all
+```
+
+Then, create a tap interface (to which RIOT will connect):
+
+```
+$ sudo ip tuntap add tap0 mode tap user ${USER}
+$ sudo ip link set tap0 up
+```
+
+Run the resulting RIOT binary by invoking:
+
+```
+$ make term
+```
+
+The application is now listening on all it's configured IP addresses.
+
+Now find out its link\_layer address:
+
+
+```
+$ make term
+/home/aabadie/riot/examples/nanocoap_server/bin/native/nanocoapcoap_server.elf tap0
+RIOT native interrupts/signals initialized.
+LED_GREEN_OFF
+LED_RED_ON
+RIOT native board initialized.
+RIOT native hardware initialization complete.
+
+main(): This is RIOT! (Version: 2015.12-devel-632-g8f451-booze-master)
+RIOT nanocoap example application
+Waiting for address autoconfiguration...
+Configured network interfaces:
+Iface  5   HWaddr: 96:3c:18:1e:26:f7
+
+           MTU:1500  HL:64  RTR  RTR_ADV
+           Source address length: 6
+           Link type: wired
+           inet6 addr: ff02::1/128  scope: local [multicast]
+           inet6 addr: fe80::e42a:1aff:feca:10ec/64  scope: local
+           inet6 addr: ff02::1:ffca:10ec/128  scope: local [multicast]
+           inet6 addr: ff02::2/128  scope: local [multicast]
+           inet6 addr: 2001:db8:1:0:e42a:1aff:feca:10ec/64  scope: global
+```
+
+The link-layer address in this case is "fe80::f8bf:2bff:fe01:9ea3", the only
+"scope: local" address set.
+
```

Comment:
Thanks for spotting this, I pushed an update. Note that this issue is also present in the microcoap example. I'll open another PR to fix it there too.